frontend/package.json

```diff
@@ -3,11 +3,18 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@google/model-viewer": "^3.2.1",
```

Comment:
There are some redundant dependencies added here that cause conflicts when installing. Adding only `@google/model-viewer` works fine for me.